homeassistant/components/onewire/quality_scale.yaml

```diff
@@ -0,0 +1,126 @@
+rules:
+  ## Bronze
+  config-flow:
+    status: todo
+    comment: Under review
+  test-before-configure: done
+  unique-config-entry:
+    status: done
+    comment: unique ID is not available, but duplicates are prevented based on host/port
+  config-flow-test-coverage:
+    status: todo
+    comment: Under review
```

Comment:
I would personally prefer to split the happy flow and the not happy flow in `test_user_flow`

Reply:
> I would personally prefer to split the happy flow and the not happy flow in `test_user_flow`

Added to status comments